packages/firestore/test/unit/local/query_cache.test.ts

```diff
@@ -51,6 +52,50 @@ describe('IndexedDbQueryCache', () => {
   });
 
   genericQueryCacheTests(() => persistencePromise);
+
+  it('persists metadata across restarts', async () => {
+    await (await persistencePromise).shutdown(/* deleteData= */ true);
+    const testPrefix = 'test-queryCache-restarts';
+    const db1 = await persistenceHelpers.testIndexedDbPersistence(
+      {},
+      testPrefix
```

Comment:
Actually, upon further thought, do we need to specify a custom prefix here? Isn't `{ dontPurgeData: true }` enough?

Reply:
Yeah, I tried that initially but must've messed something up because it nows appears that it works.